### PR TITLE
MONIT-26424 - Support OTLP Cumulative Sum metrics

### PR DESCRIPTION
**Pull request by thepeterstone**

### CODE REVIEW

**Review by keep94 — COMMENTED** [review]
Are we sure that wavefront timestamps are in nanoseconds?

**Comment by thepeterstone** [comment]
@keep94 we expect Wavefront timestamps to be in milliseconds (for instance, see [here](https://github.com/thepeterstone/wavefront-proxy/pull/3#pullrequestreview-895713498)) - but based on the Wavefront [docs](https://docs.wavefront.com/wavefront_data_format.html#metrics-data-format-syntax) that may be incorrect. I'll follow up.

File: proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufPointUtils.java [hunk 1]
```diff
@@ -42,6 +40,15 @@
...
+    return Metric.newBuilder()
+        .setName("test")
+        .setSum(Sum.newBuilder()
+            .setAggregationTemporality(AggregationTemporality.AGGREGATION_TEMPORALITY_CUMULATIVE)
+            .addAllDataPoints(points)
+            .build());
+  }
+
   public static void exportToWavefront(ExportMetricsServiceRequest request,
                                        ReportableEntityHandler<ReportPoint, String> handler,
```
[the remark this answers is not in the document]
Comment (thepeterstone) [reply proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufPointUtils.java:53]: Good point - @oppegard do we have a standard for javadocs? I'm happy to add one here.

[the remark this answers is not in the document]
Comment (oppegard) [reply proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufPointUtils.java:38]: I'm not aware of a javadoc standard for the proxy codebase. I'm sure docs are appreciated, but I haven't been adding them to public methods.

**Comment by thepeterstone** [comment]
Elsewhere in the proxy we send milliseconds (see [here](https://github.com/wavefrontHQ/wavefront-proxy/blob/851ee9c5226fe5e03c620292f88f8a194a278a2b/proxy/src/main/java/com/wavefront/agent/handlers/InternalProxyWavefrontClient.java#L90-L105)). I've confirmed that while seconds are the preferred unit, milliseconds are supported by the wavefront backend, so I'll keep the current units for consistency.

**Review by oppegard — APPROVED** [review]
lgtm

**Pull request merged**